infra/object-definitions/devspaces/instance.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: org.eclipse.che/v2
+kind: CheCluster
+metadata:
+  name: devspaces
+  namespace: openshift-devspaces
+spec:
+  devEnvironments:
+    defaultEditor: che-incubator/che-code/insiders
+    defaultNamespace:
+      template: <username>-devspaces
+    secondsOfInactivityBeforeIdling: -1
+    secondsOfRunBeforeIdling: -1
+    storage:
+      pvcStrategy: per-user
+  components:
+    cheServer:
+      debug: false
+      logLevel: INFO
+    dashboard: {}
+    database:
```

Comment:
Just curious does this actually create a DB?

Reply:
It does afaik.  I took this object definition from Raff's demo, so I would have to dig more into it to say for certain